test/JDBC/upgrade/15_5/schedule

```diff
@@ -485,6 +485,7 @@ cast_eliminate
 TestDatatypeAggSort
 babel_index_nulls_order
 BABEL-2999
+drop_index-before-15_6-or-16_1
```

Comment:
I think we missed to include the original test `drop_index` in `master/schedule` file in previous PR so let's include it this time.

Reply:
Correct, it is there in `latest/schedule` file but I am talking about `test/JDBC/upgrade/master/schedule` file.